chrome/content/zotero/elements/attachmentAnnotationsBox.js

```diff
@@ -96,16 +96,27 @@
 			if (!this.initialized || !this.item?.isFileAttachment()) return;
 			if (this._isAlreadyRendered()) return;
 
-			await Zotero.PDFWorker.renderAttachmentAnnotations(this.item.id);
-
 			this._body.replaceChildren();
 
 			if (!this._section.open || this._annotationItems.length === 0) {
 				return;
 			}
 
 			this.hidden = false;
+			let imageAnnotationRendered = false;
 			for (let annotation of this._annotationItems) {
+				if (!imageAnnotationRendered
+					&& annotation.annotationType === 'image'
+					&& !await Zotero.Annotations.hasCacheImage(annotation)) {
+					try {
+						await Zotero.PDFWorker.renderAttachmentAnnotations(annotation.parentID);
+						imageAnnotationRendered = true;
+					}
+					catch (e) {
+						Zotero.debug(e);
```

Comment:
Log levels don't really matter, but `Zotero.debug(e, 1)` if we're really throwing and `Zotero.logError(e)` if we're continuing.